### PR TITLE
Autotune doc fix

### PR DESCRIPTION
**Pull request by ybizeul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,6 +297,331 @@ room in case of sensor failure. The value should be a float between 0.0 and 100.
...
+update received from sensor after this time period, the system considers the sensor as stall and 
+switch to safety mode, the output being forced to output_safety. If set to 0, the feature is 
+disabled. Can be float in seconds or time hh:mm:ss (default 6 hours).
+* **output_safety** (Optional): Sets the output level of the PID once the thermostat enters safety 
+mode due to unresponsive temperature sensor. This can help to keep a minimum temperature in the 
+room in case of sensor failure. The value should be a float between 0.0 and 100.0 (default 5.0).
+* **initial_hvac_mode** (Optional): Forces the operation mode after Home Assistant is restarted. If 
+not specified, the thermostat will restore the previous operation mode.
+* **debug** (Optional): Make the climate entity expose the following internal values as extra 
+states attributes, so they can be accessed in HA with sensor templates for debugging purposes (
```
Comment (Chupaka) [inline README.md:624]: Why is this duplicated? :)

**Comment by ybizeul** [comment]
Well, that'one one messy PR !

**Review by ScratMan — APPROVED** [review]

**Comment by Chupaka** [comment]
@ScratMan omg, three messy commits in the master branch?.. Why not squash them into a single one?..

**Comment by ScratMan** [comment]
> @ScratMan omg, three messy commits in the master branch?.. Why not squash them into a single one?..

Merging on the Valentine's day... Had no time to make it clean 😅

**Pull request merged**